doc/source/serve/multi-app.md

```diff
@@ -0,0 +1,214 @@
+
+### Application and its deployments
+Click into the first application, `app1` to see an overview of the application:
+
+![application-overview](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/multi-app/application-overview-dashboard.png)
+
+It also shows the list of deployments under that application:
+
+![deploymentsd](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/multi-app/deployments-dashboard.png)
+
+### Replicas
+Click the drop down arrow for the first deployment, `Adder` to see the list of replicas for that deployment:
+
+![replicas](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/multi-app/replica-dashboard.png)
+
+Click into one of the replicas to see an overview of the replica as well as a list of actor tasks run on the replica actor:
+
+![replica-overview](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/multi-app/replica-overview-dashboard.png)
+
+## New Multi-Application Config
+
+Use the config from the above tutorial as an example. In a multi-application config, the first section is for cluster-level config options:
+```yaml
+proxy_location: EveryNode
+http_options:
+  host: 0.0.0.0
+  port: 8000
+```
+
+Then, specify a list of applications to deploy to the Ray cluster. Each application must have a unique name and route prefix.
+```yaml
+applications:
+
+- name: app1
+  route_prefix: /calculator
+  import_path: calculator:app
+  runtime_env: {}
+  deployments:
+  - name: Multiplier
+  - name: Adder
+  - name: Router
+  - name: DAGDriver
+
+- name: app2
+  route_prefix: /greet
+  import_path: greet:app
+  runtime_env: {}
+  deployments:
+  - name: greet
+  - name: DAGDriver
+```
+
+(serve-config-migration)=
+### Migrating from a Single-Application Config
+
+Migrating the single-application config `ServeApplicationSchema` to the multi-application config format `ServeDeploySchema` is straightforward. Each entry under the  `applications` field matches the old, single-application config format. To convert a single-application config to the multi-application config format:
+* Copy the entire old config to an entry under the `applications` field.
+* Remove `host` and `port` from the entry and move them under the `http_options` field.
+* Name the application.
+* If you haven't already, set the application-level `route_prefix` to the route prefix of the ingress deployment in the application.
+* When needed, add more applications.
+
+For more details on the multi-application config format, see the documentation for [`ServeDeploySchema`](serve-rest-api-config-schema).
+
+:::{note} 
+You must remove `host` and `port` from the application entry. In a multi-application config, specifying cluster-level options within an individual application isn't applicable, and is not supported.
+:::
+
+## One application vs Multiple Applications
```

Comment:
Imo this should be at the top of the page "When to use multiple applications"

Reply:
Moved and renamed!